components/allowances/dashboard/columns.tsx

```diff
@@ -171,6 +173,14 @@ export const columns = [
     sortingFn: customSortingFns.timestamp,
     sortUndefined: 1,
   }),
+  columnHelper.accessor('lastCancelled', {
```

Comment:
We don't need to include "Last Cancelled" in the "approvals" table (under the "Approvals" tab). We only need to include it in the "permit signatures" table (under the "Signatures" tab). So that means that we do not need to list it here in the `columns` array.